### PR TITLE
TASK: Run post-update scripts only once 

### PR DESCRIPTION
**Pull request by daniellienert**
Previously the scripts were run for every package at the end of the install process.
The scripts only need to be run once.

### CODE REVIEW

**Comment by daniellienert** [comment]
See https://github.com/neos/flow-development-collection/pull/2507/commits/d79947442d7de8c59f0d6460e0affef30406148b for the pure code change without code cleanup.

**Comment by albe** [comment]
😍 this was soooo long on my "this can't be right, need to take a look sometime" list!

File: Neos.Flow/Classes/Composer/InstallerScripts.php [hunk 1]
```diff
@@ -56,17 +77,25 @@ public static function postUpdateAndInstall(Event $event)
...
      * @param PackageEvent $event
      * @return void
+     * @throws Exception\InvalidConfigurationException
      * @throws Exception\UnexpectedOperationException
+     * @throws \Neos\Utility\Exception\FilesException
      */
-    public static function postPackageUpdateAndInstall(PackageEvent $event)
+    public static function postPackageUpdateAndInstall(PackageEvent $event): void
     {
+        if (self::$postPackageUpdateAndInstallAlreadyRun) {
```
Comment (albe) [inline Neos.Flow/Classes/Composer/InstallerScripts.php:95]: Just one question, which I'm not sure about: Is this script run once after each installed/updated package, or once for each package after they are all installed/updated?

Comment (daniellienert) [reply Neos.Flow/Classes/Composer/InstallerScripts.php:95]: The script is triggered when all packages are installed updated for every of that packages

**Review by albe — APPROVED** [review]

**Review by fcool — APPROVED** [review]

**Pull request merged**